src/app/select-store/page.tsx

```
@@ -1,109 +1,72 @@
+            <p className="text-gray-500">
+              Choisissez la boutique avec laquelle vous souhaitez travailler
+            </p>
+          </div>
+
+          <div className="grid gap-4">
+            {user.stores.map((store) => (
+              <div
+                key={store.id}
+                onClick={() => handleStoreSelect(store)}
+                className="p-6 border rounded-lg hover:border-primary hover:bg-primary/5 cursor-pointer transition-all group"
               >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)